modules/@apostrophecms/image/ui/apos/components/AposMediaManager.vue

```diff
@@ -113,8 +115,14 @@ export default {
     options() {
       return window.apos.modules[this.moduleName];
     },
+    moduleLabels() {
+      return {
+        singular: this.options.label,
+        plural: this.options.pluralLabel
+      };
+    },
     selected() {
-      return this.media.filter(item => this.checked.includes(item._id));
+      return this.items.filter(item => this.checked.includes(item._id));
```

Comment:
Based on my reading of how this computed prop is used to populate the selection list, it would lose its page 1 contents if the user moves to page 2.

Reply:
I didn't change this here. This is simply renaming. I think what you're talking about is resolved in #2420

Reply:
It depends how the relationship properties are managed, but those should be in the object I'm storing in #2420